.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+**Checklist**
+Please make sure you have finished the following tasks before opening the PR.
+
+- [ ] Please read [Contributing to QuTiP Development](https://github.com/qutip/qutip-doc/blob/master/qutip_dev_contrib.md)
+- [ ] Contributions to qutip should follow the [pep8 style](https://www.python.org/dev/peps/pep-0008/).
+You can use [pycodestyle](http://pycodestyle.pycqa.org/en/latest/index.html) to check your code automatically
+
+Delete this section after you have complete all the tasks.
+
+**Description**
+Describe here the proposed change.
+
+**Related issues or PRs**
+Please mention the related issues or PRs here. If the PR fixes an issue, use the keyword fix/fixes/fixed followed by the issue id, e.g. fix #1184
+
+**Changelog**
+Give a short description of the PR in a few words. This will be shown in the QuTiP change log after the PR gets merged.
```

Comment:
If we want them to update the Changelog, this should be more explicit. I feel like the Changelog in `qutip-doc` puts a lot of strain for people to comply, as they have to fork and open a PR also there. My proposal is to add a `changelog` file also in `qutip/qutip` that can be copy pasted every time there is a new release also in `qutip-doc`.  What do you think? Can you add such file? It could be named the same or something like `changelog-PRs`, I don't know.